src/main/java/org/esbtools/gateway/resync/ResyncError.java

```diff
@@ -0,0 +1,13 @@
+package org.esbtools.gateway.resync;
+
+public final class ResyncError {
+
+    public static final String ALL_REQUIRED_VALUES_NOT_PRESENT = "One or more required values was not present";
+    public static final String SYSTEM_NOT_CONFIGURED = "The %s system is not configured properly";
+    public static final String PROBLEM_ENQUEUING = "There was a problem enqueuing the selected message: %s";
+
+    public static String withContext(String errorMessage, String variableValue) {
+        return String.format(errorMessage, variableValue);
+    }
```

Comment:
Another better example would be you can also make them type safe so kind of intention revealing, so for the other error message:

```java
public static String problemEnqueing(ResyncRequest request) {
  return "There was a problem enqueuing the selected message: " + request;
}
```

So now it's clear that for this error what you intend to include is the request with the error.